internal/constants/constants.go

```diff
@@ -0,0 +1,50 @@
+package constants
+
+import (
+	"os"
+
+	"log/slog"
+)
+
+const (
+	// DefaultAppFolder is the name of the default root folder
+	DefaultAppFolder = "ubuntu-insights"
+
+	// DefaultLogLevel is the default log level selected without any verbosity flags
+	DefaultLogLevel = slog.LevelInfo
+)
+
+type options struct {
+	baseDir func() (string, error)
+}
+
+type option func(*options)
+
+// GetDefaultConfigPath is the default path to the configuration file
+func GetDefaultConfigPath(opts ...option) string {
+	o := options{baseDir: os.UserCacheDir}
```

Comment:
This default appears flipped. `GetDefaultConfigPath` returns `os.UserCacheDir` by default while `GetDefaultCachePath` return `os.UserConfigDir` by default